### ISSUE TITLE
Please link GitHub locations

### ISSUE THREAD

**Issue by mkarg** [mvc-spec/mvc-spec#84]
It is very confusing that mvc-spec is not part of the javaee organization of GitHub, and even worse, that there are two instances of the MVC source code found on GitHub (one here, one there) and no word of clarification which is the current one.

I would kindly like to propose to ask the spec leads to discuss with the javaee organization if they could replace their source code by link into ours, or vice versa ask them to make the MVC spec leads master of their source code repository so we can remove ours?

Actually I think the correct way would be that javaee only maintains forks of the standalone specs, so they should remove their repo and replace it with a fork of ours.

**Comment by chkal** [comment]
I agree that the current situation is confusing. But please note that we are still in the process of migrating the infrastructure. That's why there hasn't been any official statement about this on the mailing list yet. I'm sorry that it takes so long.

About the current status: Our plan was to keep the mvc-spec organization as the place where we do all the work. That's why I migrated the JIRA issues to the corresponding repositories (mvc-spec/mvc-spec and mvc-spec/ozark) about a week ago. In the mean time Oracle also migrated the mvc-spec repository to their javaee organization. To our surprise they also migrated the JIRA issues. Unfortunately we don't have admin permissions for the javaee/mvc-spec repository yet. As soon as we get the required permissions, we will most likely disable the issue tracker there. As soon as this is done, there will be an official statement on the mailing list.

To sum up: We are sorry that the transition is taking so long. But I'm confident, that we can finish it early next week. So stay tuned.

**Comment by mkarg** [comment]
Christian, thanks a lot for the status update! Actually my intention was _not_ to trouble you in any way. It is simply Oracle's fault. I just wanted to say that Ivar and you could ask the Oracle guy that is in charge of the GitHub transition to remove his MVC repo and then fork ours.

**Comment by ivargrimstad** [comment]
Markus, it's not Oracle's fault. I opted in on their offer to migrate the repos from java.net to github so we are still in the good companionship of the other Java EE specifications. The minor confusion of having double repos right now was expected, but I wanted the migration to be done first, and then handle that issue... It will be all right! :)

**Comment by ivargrimstad** [comment]
The issues on javaee/mvc-spec are now closed with the message "Duplicate of [the corresponding issue on mvc-spec/mvc-spec]".
I will check how we can, in the easiest way, link the source code as well.

**Comment by chkal** [comment]
@mkarg: I didn't get this wrong. No worries. I just wanted to explain the current situation and what happend behind the scenes.

@ivargrimstad: Great! Thanks!

**Comment by mkarg** [comment]
@ivargrimstad Great! Thanks a lot! :-)

**Comment by Daniel-Dos** [comment]
I agree that it was confusing to have duplicated repo's specification, concluding that it was due to migration with Java.net as it has been lately.
But the explanation of @ivargrimstad and @ chkal was very enlightening.

Thank you so much.
